includes/Helpers/ManageWikiPermissions.php

```diff
@@ -179,6 +195,52 @@ public function commit() {
 				);
 
 				$this->deleteUsersFromGroup( $group );
+			} elseif ( array_key_exists( $group, $this->renameGroups ) ) {
+				$this->log = 'rename';
```

Comment:
This should all be part of rename

Reply:
But then if that's all moved to `rename()` then wouldn't `if ( empty( $this->livePermissions[$group]['permissions'] ) )` trigger and return an `emptygroup` error?